.github/workflows/sphinx.yml

```diff
@@ -7,21 +7,26 @@ on:
     branches:
       - master
 
+env:
+  POETRY_VERSION: "1.3.1"
+  PYTHON_VERSION: "3.9"
+
 jobs:
   build-docs:
-    runs-on: ubuntu-latest
+    runs-on: "ubuntu-latest"
     steps:
       - name: Checkout
-        uses: actions/checkout@master
+        uses: actions/checkout@v3
         with:
           fetch-depth: 0
 
-      - uses: actions/setup-python@v2
+      - uses: actions/setup-python@v4
         with:
-          python-version: 3.9
-
-      - uses: abatilo/actions-poetry@v2.1.3
+          python-version: ${ env.PYTHON_VERSION }
```

Comment:
I think we need double curly braces instead of single curly braces here. Single braces doesn't seem to parse the environment variable properly.

Reply:
Change made in 5ab2a7d.